recipe/session/jwt.go

```diff
@@ -25,12 +25,12 @@ const header = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsInZlcnNpb24iOiIyIn0="
 func verifyJWTAndGetPayload(jwt string, jwtSigningPublicKey string) (map[string]interface{}, error) {
 	var splitted = strings.Split(jwt, ".")
 	if len(splitted) != 3 {
-		return nil, errors.GeneralError{
+		return nil, errors.BadInputError{
```

Comment:
These should be regular errors.. `BadInputError` are only for API input checking errors